src/app/api.mjs

```diff
@@ -166,10 +167,22 @@ export async function getChildren() {
 }
 
 export async function getBookings(dateAfter, dateBefore) {
-  const response = await client.get(BOOKINGS_PATH, {
+  const url = `${BOOKINGS_PATH}/`;
+  const response = await client.get(url, {
+    params: {
+      date_after: dateAfter.format("YYYY-MM-DD"),
```

Comment:
It's a bit out of sync with how we pass data, i.e. we pass all body params within a single object, but here we pass query params as 2 separate args. I would do the same here, i.e. `export async function getBookings({dateAfter, dateBefor})`